docs/docs/0400-user-guide/350-orchestration.mdx

```diff
@@ -1,55 +1,1618 @@
+---
+# Display h2 to h5 headings
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
 # Orchestration
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 
-Now that we have seen how to load and transform data, we will see how to orchestrate the tasks using the `starlake` command line tool.
-starlake is not an orchestration tool, but it can be used to generate your DAG based on templates and to run your transforms in the right order on your tools of choice for scheduling and monitoring batch oriented workflows.
+Now that we have seen how to load and transform data, we will see how to orchestrate the tasks using the `starlake` command line tool.<br/>
+starlake is not an orchestration tool, but it can be used to generate your DAG based on templates and to run your transforms in the right
+order on your tools of choice for scheduling and monitoring batch oriented workflows.<br/>
+<br/>
+Starlake DAG generation relies on:
 
+- **starlake** command line tool
+- **DAG** **configuration**(s) and their **references** within the loads and tasks
+- **template**(s) that may be customized
+- **starlake-orchestration** framework to dynamically generate the tasks that will be run
+- **managing dependencies** between tasks to execute transforms in the **correct order**
 
-## Data loading
-Templates may be defined in the following directories:
-- an absolute path name
-- a relative path name to the `metadata/dags/templates/` directory
-- a relative path name to the `src/main/resources/templates/dags/` resource directory
+## Prerequisites
 
-Below a few DAG templates available in the resource directory:
+Before using Starlake dag generation, ensure the following minimum versions are installed on your system:
 
-|DAG Template | Description |
-|---------|-------------|
-`load/schedule_table_bash.py.j2` | This template executes individual bash jobs and requires the following dag generation options set: <br/> - SL_ROOT: The root project path <br/> - SL_STARLAKE_PATH: the path to the starlake executable |
-`load/schedule_table_cloudrun.py.j2` | This template executes individual cloud run jobs and requires the following dag generation options set: <br/> - SL_ROOT: The root project path <br/> - SL_STARLAKE_PATH: the path to the starlake executable |
-`load/sensor_table_cloudrun_with_ack.py.j2`| This template executes individual cloud rub jobs upon ack file detection and requires the following dag generation options set: <br/> - SL_ROOT: The root project path <br/> - SL_STARLAKE_PATH: the path to the starlake executable |
-`load/single_scheduled_tasks_with_wait_sensor.py.j2` | This template executes individual cloud run upon file detection and requires the following dag generation options set: <br/> - SL_ROOT: The root project path <br/> - SL_STARLAKE_PATH: the path to the starlake executable |
+- **starlake**: 1.0.1 or higher
+- **python**: 3.8 or higher
```

Comment:
Python is not a prerequisite of the dag generation feature. It is a prerequisite of the Orchestrator

Reply:
python prerequisite removed